src/queries/SearchResultQuery.ts

```diff
@@ -30,6 +30,7 @@ class SearchResultQuery extends BaseQuery<SearchResult> implements SearchResultQ
     this.buildKeywordQueryParams();
     this.buildUrlQueryParams();
     await this.buildUrlWithWordQueryParams();
+    await this.buildUrlWithAtLeastCharactersQueryParams();
     this.buildPaginationQueryParams();
```

Comment:
Let's have a line break to separate the filter builders and the pagination builders 💪 

```suggestion
    this.buildKeywordQueryParams();
    this.buildUrlQueryParams();
    await this.buildUrlWithWordQueryParams();
    await this.buildUrlWithAtLeastCharactersQueryParams();
    
    this.buildPaginationQueryParams();
```